hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -176,6 +209,11 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 		return reconcile.Result{}, fmt.Errorf("error validating autoscaling parameters: %w", err)
 	}
 
+	if hcluster.Status.IgnitionEndpoint == "" {
+		r.Log.Info("Ignition endpoint not available, requeuing")
+		return reconcile.Result{Requeue: true}, nil
```

Comment:
Not sure how long this generally takes to stand up... worth doing a `RequeueAfter` to avoid pushing up against the rate limiter if we know it'll be several seconds?